### PR TITLE
Added cordova 6 tools release blog post

### PR DESCRIPTION
**Pull request by stevengill**
Please review this blog post for the cordova 6 release!

### CODE REVIEW

File: www/_posts/2016-01-27-tools-release.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+* [cordova-lib@6.0.0](https://www.npmjs.org/package/cordova-lib)
+* [cordova@6.0.0](https://www.npmjs.org/package/cordova)
+* [plugman@1.1.0](https://www.npmjs.org/package/plugman)
+* [cordova-js@4.1.3](https://www.npmjs.org/package/cordova-js)
+
+Release Highlights include:
+* Updated default platform versions to `cordova-android@5`, `cordova-ios@4` and `cordova-windows@4.3`. 
+    - `cordova-android@5` supports **Android marshmallow** 
+    - `cordova-ios@4` adds **iOS9** and `wkwebview` support
```
Comment (dpogue) [inline www/_posts/2016-01-27-tools-release.md]: `WKWebView`

File: www/_posts/2016-01-27-tools-release.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    - `cordova-windows@4` updated the platform to use the new `PlatformAPI`
```
Comment (dpogue) [inline www/_posts/2016-01-27-tools-release.md]: Platform API without backticks? Is Platform API considered the name of an API itself, or the name of a collection of APIs?

File: www/_posts/2016-01-27-tools-release.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+* Added a new template feature to `create` command. E.g. `cordova create --template cordova-app-hello-world`. This can pull templates from **npm** and **git urls**.
+* Removed support for our old **Cordova Plugins Registry**. Now plugins can only be installed via **npm**, **gitURL** or local path.
```
Comment (dpogue) [inline www/_posts/2016-01-27-tools-release.md]: **git URL**?

File: www/_posts/2016-01-27-tools-release.md (hunk 2 above)
Comment (stevengill) [inline www/_posts/2016-01-27-tools-release.md]: Makes sense to remove the backticks

File: www/_posts/2016-01-27-tools-release.md (hunk 3 above)
Comment (stevengill) [inline www/_posts/2016-01-27-tools-release.md]: I'll correct all instances to be **git URL**

File: www/_posts/2016-01-27-tools-release.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+* [cordova-lib@6.0.0](https://www.npmjs.org/package/cordova-lib)
+* [cordova@6.0.0](https://www.npmjs.org/package/cordova)
+* [plugman@1.1.0](https://www.npmjs.org/package/plugman)
+* [cordova-js@4.1.3](https://www.npmjs.org/package/cordova-js)
+
+Release Highlights include:
+* Updated default platform versions to `cordova-android@5`, `cordova-ios@4` and `cordova-windows@4.3`. 
+    - `cordova-android@5` supports **Android marshmallow** 
+    - `cordova-ios@4` adds **iOS9** and `WKWebView` support
+    - `cordova-windows@4` updated the platform to use the new Platform API.
```
Comment (nikhilkh) [inline www/_posts/2016-01-27-tools-release.md]: While technically, not incorrect - I would recommend providing specific major versions here too especially for Windows & iOS - 4.3 and 5.1

**Comment by nikhilkh** [comment]
LGTM

File: www/_posts/2016-01-27-tools-release.md (hunk 4 above)
Comment (stevengill) [inline www/_posts/2016-01-27-tools-release.md]: I take it you mean windows & android not ios. 

I'll update that. 

Also, on line 44, I point to android 5.0 blog post but i'll update that to point to 5.1 blog post instead

**Pull request merged**